01week/resume/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>HB Resume</title>
+ <link rel="stylesheet" type="text/css" media="screen" href="./css/style.css">
+</head>
+<!-- command forward slash -->
+<body>
+    <!-- left side of webpage -->
+    <div class="row">
```

Comment:
I would like to see a bit more use of semantic tags. Example, your class row can be <main> then the columns can be <section>.

Reply:
Okay, thank you and should I add those into my page and re-commit as final.2?

Reply:
Perfect, that's what I'm going to do! I'll have it done tomorrow!

Reply:
Or is it too late? I see that this has been merged into gh-pages today.